pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/metadata-step/metadata-step.component.ts

```diff
@@ -36,48 +46,85 @@ export class MetadataStepComponent extends StepFormDirective implements OnInit {
             const savedLabelsArray = savedLabelsString.split(', ')
             savedLabelsArray.map(label => {
                 const labelArray = label.split(':');
-                this.labels.set(labelArray[0], labelArray[1]);
+                this.addLabel(labelArray[0], labelArray[1]);
             });
         }
         super.initFormWithSavedData();
+        this.formGroup.valueChanges.pipe(
```

Comment:
I think we can use the 'registerOnValueChange' method that is in step-form.ts for this

@swalner-vmware can you confirm that would be the correct usage pls?

Reply:
Yes, this would be a good use for registerOnValueChange

Reply:
I also missed that this is a formGroup.valueChanges....

Reply:
fixed